packages/xo-web/src/common/selectors.js

```diff
@@ -273,13 +273,13 @@ const _getPermissionsPredicate = invoke(() => {
     }
   )
 
-  return state => {
+  return (state, props, useResourceSet) => {
```

Comment:
Is this necessary? Didn't we find a way to get resolved resource sets for Self users in the Self view?

Reply:
No,  for  `selectors/getResolvedResourceSets`, the `createGetObjectsOfType` doesn't get objects for self view because `createGetObjectsOfType` uses  [`_getPermissionsPredicate`](https://github.com/vatesfr/xen-orchestra/blob/master/packages/xo-web/src/common/selectors.js#L247toL289) that doesn't take into account the resourceSet.

Reply:
True :/